hoot-core/src/main/cpp/hoot/core/conflate/network/NetworkDetails.cpp

```diff
@@ -336,7 +336,7 @@ double NetworkDetails::calculateStringLocation(ConstEdgeStringPtr es, ConstEdgeL
 
   // if el1 is not part of this string, but shares a vertex with the first edge then we can
   // still calculate and return the distance.
-  if (el->isExtreme() && el->getVertex() == edgeStartVertex)
+  if (el->isExtreme() && el->getVertex() == edgeEndVertex)
```

Comment:
This change might require some further inspection.  Here is how I came to the change.  Above there is code to set `edgeStartVertex` and then uses it in a very similar `if` conditional to the original (see lines 299-313).  Then in lines 328-335 `edgeEndVertex` is set and not used.  It appears as though this should be `edgeEndVertex` instead of `edgeStartVertex` here.  In the end, no unit test results changed so I don't know if it made a difference but it now looks correct to me.

Reply:
I think that looks like a correct change for this.  Keep an eye on the network regression tests anyway...I'm not sure how solid network unit test coverage is (never checked)